Policies/github-corporate-terms-of-service.md

```diff
@@ -37,13 +37,13 @@ Effective date: May 22, 2017
 4. "Customer", “You,” and “Your” refer to the company, or organization that has visited or is using the Website or Service; that accesses or uses any part of the account; or that directs the use of the account in the performance of its functions. Special terms may apply for business or government accounts (See [Section B(6): Additional Terms](#6-additional-terms)).
-5. "User" refers to the individual who has visited or is using the Website or Service on your behalf; that accesses or uses any part of the account on your behalf; or that directs the use of the account in the performance of their functions on your behalf. A User must be at least 13 years of age.
+5. "User" refers to the individual who has visited or is using the Website or Service on your behalf; that accesses or uses any part of the account on your behalf; or that directs the use of the account in the performance of their functions on your behalf. 
 6. “GitHub,” “We,” and “Us” refer to GitHub, Inc., as well as our affiliates, directors, subsidiaries, contractors, licensors, officers, agents, and employees.
 7. A "Corporate Account" refers to any account created by a User on behalf of an entity, such as a company, non-profit organization, or group. A Corporate Account may include a Business plan, a Team plan, or a Developer plan.
 8. “Content” refers to content featured or displayed through the Website, including without limitation text, data, articles, images, photographs, graphics, software, applications, designs, features, and other materials that are available on the Website or otherwise available through the Service. "Content" also includes Services. “User-Generated Content” is Content, written or otherwise, created or uploaded by our Users. “Paid Content” is Content only available to Users who are participating in a payment plan, including private repositories.
 
 ### B. Account Terms
-**Short version:** *A human must create your account; the creator and Users of your account must be 13 or over; you must provide a valid email address; and you may not have more than one free account. You alone are responsible for your account and anything that happens while you are signed in to or using your account. You are responsible for keeping your account secure.*
+**Short version:** *A human must create your account; the creator of your account must be 24 or over; you must provide a valid email address; and you may not have more than one free account. You alone are responsible for your account and anything that happens while you are signed in to or using your account. You are responsible for keeping your account secure.*
```

Comment:
That seems arbitrary at best. Make it 18 -- the age of 13 is to comply wth COPPA... so I'm :-1:  on this.